File: pos-android/create_logo.js
const { createCanvas } = require('canvas');
const fs = require('fs');

// Ukuran canvas
const width = 200;
const height = 80;

// Buat canvas dengan background transparan
const canvas = createCanvas(width, height);
const ctx = canvas.getContext('2d');

// Warna - RGB format
const liteColor = '#0f172a';  // Biru Tua
const posColor = '#3b82f6';   // Biru Benhur

// Font
ctx.font = 'bold 50px Arial';

// Gambar teks 'Lite'
ctx.fillStyle = liteColor;
ctx.fillText('Lite', 10, 55);

// Gambar teks 'POS'
ctx.fillStyle = posColor;
ctx.fillText('POS', 110, 55);

// Simpan PNG transparan
const pngBuffer = canvas.toBuffer('image/png');
fs.writeFileSync('D:/AndroidPos/src/assets/logo.png', pngBuffer);
console.log('Logo PNG saved: D:/AndroidPos/src/assets/logo.png');

// Buat versi untuk struk - background putih
const canvasReceipt = createCanvas(width, height);
const ctxReceipt = canvasReceipt.getContext('2d');

// Background putih
ctxReceipt.fillStyle = '#ffffff';
ctxReceipt.fillRect(0, 0, width, height);

// Gambar teks
ctxReceipt.font = 'bold 50px Arial';
ctxReceipt.fillStyle = liteColor;
ctxReceipt.fillText('Lite', 10, 55);
ctxReceipt.fillStyle = posColor;
ctxReceipt.fillText('POS', 110, 55);

// Simpan JPG
const jpgBuffer = canvasReceipt.toBuffer('image/jpeg', { quality: 0.95 });
fs.writeFileSync('D:/AndroidPos/src/assets/receipt_logo.jpg', jpgBuffer);
console.log('Receipt logo saved: D:/AndroidPos/src/assets/receipt_logo.jpg');

File: pos-android/create_logo.py
import base64
from PIL import Image, ImageDraw, ImageFont

# Buat gambar baru dengan background transparan
img = Image.new('RGBA', (200, 80), (255, 255, 255, 0))
draw = ImageDraw.Draw(img)

# Warna - RGB format
lite_color = (15, 23, 42)   # #0f172a - Biru Tua
pos_color = (59, 130, 246)  # #3b82f6 - Biru Benhur

# Gunakan font system
try:
    font = ImageFont.truetype('arial.ttf', 50)
except:
    font = ImageFont.load_default()

# Gambar teks
draw.text((10, 10), 'Lite', fill=lite_color, font=font)
draw.text((110, 10), 'POS', fill=pos_color, font=font)

# Simpan PNG transparan
output_path = 'D:/AndroidPos/src/assets/logo.png'
img.save(output_path, 'PNG')

# Buat versi untuk struk - background putih
img_receipt = Image.new('RGB', (200, 80), (255, 255, 255))
draw_receipt = ImageDraw.Draw(img_receipt)
draw_receipt.text((10, 10), 'Lite', fill=lite_color, font=font)
draw_receipt.text((110, 10), 'POS', fill=pos_color, font=font)

receipt_path = 'D:/AndroidPos/src/assets/receipt_logo.jpg'
img_receipt.save(receipt_path, 'JPEG', quality=95)

print("Logo PNG:", output_path)
print("Receipt logo:", receipt_path)